src/components/ExploreSpec.js

```diff
@@ -148,31 +116,51 @@ class ExploreSpec extends React.Component {
                 </div>
-                : categoryToShow === "Cluster information" ?
-                  <ExploreInfo name={"Cluster information"} infos={clusterInfo} isMobile={isMobile} />
-                  : categoryToShow === "Database" ?
-                    <ExploreInfo name={"Database"} infos={databaseInfo} isMobile={isMobile} />
-                    : categoryToShow === "Storage" ?
-                      <ExploreInfo name={"Storage"} infos={storageInfo} isMobile={isMobile} />
-                      : categoryToShow === "Networking" ?
-                        <ExploreInfo name={"Networking"} infos={networkingInfo} isMobile={isMobile} />
-                        : categoryToShow === "Performance" ?
-                          <ExploreInfo name={"Performance"} infos={performanceInfo} isMobile={isMobile} />
-                          : categoryToShow === "Cloud provider" ?
-                            <ExploreInfo name={"Cloud provider"} infos={cloudProviderInfo} isMobile={isMobile} />
-                            : categoryToShow === "External app dependecies" ?
-                              <ExploreInfo name={"External app dependecies"} infos={externalAppInfo} isMobile={isMobile} />
-                              : null
+              }
+              {categoryToShow === "" && tagToShow === "" ?
+                specJson.categories.map(category => {
+                  const categorySpecs = specJson.specs.filter(spec => category.name === spec.category);
+                  return (
+                    <ExploreInfo name={category.display} specs={categorySpecs} isMobile={isMobile} />
+                  )
+                })
+                :
+                tagToShow ?
+                  <div className="Info--wrapper flex flexWrap--wrap u-marginTop--30">
+                    {specsToShow.map((spec, i) => {
+                      return (
+                        <div className="flex alignItems--center" key={`${spec.id}-${i}`}>
+                          <span className="category-icon" style={{ backgroundImage: `url(${spec.iconUri})` }} />
+                          <div className="flex-column u-marginLeft--12">
+                            <p className="u-fontSize--largest u-color--biscay u-fontWeight--bold u-lineHeight--more"> {spec.title} </p>
+                            <p className="u-fontSize--small u-color--tundora body-copy u-marginTop--8"> {spec.description} </p>
+                          </div>
+                        </div>
+                      )
+                    })}
+                  </div>
+                  :
+                  categoryToShow === "Cluster information" ?
+                    <ExploreInfo name={"Cluster information"} specs={specJson.specs.filter(spec => "cluster_information" === spec.category)} isMobile={isMobile} />
+                    : categoryToShow === "Database" ?
```

Comment:
These all feel like they could be refactored to be rendered just one time by doing something like this

```
<ExploreInfo name={categoryToShow} specs={specJson.specs.filter(spec => find(specJson.categories, ["display", categoryToShow]).name === spec.category)} isMobile={isMobile} />
```

Would this work or am I misunderstanding how this is functioning? It would be nice to cut down the receptiveness of this code if possible.